packages/abstraxion/src/hooks/useAbstraxionAccount.ts

```diff
@@ -50,7 +50,7 @@ export const useAbstraxionAccount = (): AbstraxionAccountState => {
       setIsConnecting(false);
     }
 
-    if ((!isConnecting && !abstraxionAccount) || !granterAddress) {
+    if (!isConnecting && !abstraxionAccount && !granterAddress) {
```

Comment:
Why this change? @BurntVal

Reply:
I changed it the OR in the previous PR but went back to AND because I found that it was causing an unnecessary surplus of re-renders